src/implementation/Client/GRPCClient/sidecar.ts

```diff
@@ -36,4 +37,22 @@ export default class GRPCClientSidecar implements IClientSidecar {
       });
     });
   }
+
+  static async isStarted(client: GRPCClient): Promise<boolean> {
+    const callClient = await client.getClient(false);
+
+    return new Promise((resolve, _reject) => {
+      try {
+        callClient.getMetadata(new Empty(), (err, _res: GetMetadataResponse) => {
+          if (err) {
```

Comment:
nit: unused var, both `err` and `e` below.

Reply:
e is not used, err is used in the iff check